src/_includes/components/link/link.njk

```diff
@@ -1 +1,8 @@
-<a href="https://guide.inclusivedesign.ca"{% if params.external %} rel="external"{% endif %}>{{ params.text }}</button>
+{% block head %}
+    <link rel="stylesheet" href="/static/css/link.css" />
+{% endblock %}
+
+
+<div {% if params.variantClass %} class="link-container {{ params.variantClass }}" {% endif %}>
+    <a href="#">{{ params.text }}</a>
+</div>
```

Comment:
```suggestion
{% if params.variantClass %}
<div  class="{{ params.variantClass }}" >
{% endif %}
    <a href="#">{{ params.text }}</a>
{%-if params.variantClass %}
</div>
{% endif %}
```

Reply:
I don't think we should have a special "link-container"— we should only wrap links which need a different background.

Reply:
Thanks Ned for the suggestion, I created an applied the new utility class. But it doesn't scale too well on the examples, please let me know what you think once you see them.

Reply:
Oh, I forgot to add padding, please disregard this comment.